ergo/distributions/truncated.py

```diff
@@ -0,0 +1,63 @@
+from dataclasses import dataclass
+from .distribution import Distribution
+import jax.numpy as np
+
+
+def truncate(underlying_dist_class: Distribution, floor: float, ceiling: float):
```

Comment:
I wanted to use something different from `scale_min` / `scale_max` because we might eventually want the to actually have different values here, i.e. not do `scale_min=floor`, `scale_max=ceiling`. But maybe that's silly